docker-compose/camunda-8.6/.env

```diff
@@ -1,11 +1,11 @@
 ## Image versions ##
 # renovate: datasource=docker depName=camunda/connectors-bundle
-CAMUNDA_CONNECTORS_VERSION=8.6.0-alpha2-rc2
+CAMUNDA_CONNECTORS_VERSION=8.6.0-alpha2.1
```

Comment:
For Connectors, it was `alpha2.1` before (see https://github.com/camunda/camunda-platform/pull/668).